Cargo.toml

```diff
@@ -76,6 +75,7 @@ zip = "0.6"
 
 # pin lightningcss, used by trunk, also pulled in by minify-html
 lightningcss = "=1.0.0-alpha.54"
+lol_html = "1.2.1"
```

Comment:
Move this up to the main block (sorted alphabetically). This section is just for things to call out.